sanctuary/sanctuary-types.d.ts

```diff
@@ -54,6 +54,7 @@ type NonZeroFiniteNumber = number; // - Infinity, -Infinity, 0, -0, NaN
 type ValidNumber = number; // - NaN
 
 export interface Functor<F> {
+  map<U>(fn: (f: F) => U): Functor<U>;
```

Comment:
But PR is against master, which is intended for Sanctuary 0.11. Weren't those prefixed methods added in 0.12?